recipe/meta.yaml

```diff
@@ -22,33 +22,40 @@ requirements:
 
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
     - wheel
+    - flit-core >=3.2,<4
 
   run:
-    - python >=3.6
-    - numpy >=1.15
-    - matplotlib >=2.2
-    - scipy >=1.0
-    - pandas >=0.23
+    - python
+    - numpy >=1.17
+    # matplotlib 3.6.1 incompatible with latest master for 0.12.0
+    # https://github.com/mwaskom/seaborn/issues/3072
+    - matplotlib >=3.1,!=3.6.1
```

Comment:
Can we change this pin to `matplotlib-base` instead? If we do not, this package pulls in the `qt` dependency unnecessarily

Reply:
Note that Conda Forge has adopted a multi-output model that builds `seaborn-base` and `seaborn`